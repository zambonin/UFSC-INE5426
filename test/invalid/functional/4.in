# mismatching parameter types
float t[10]
int output[10]
output = map(lambda int x -> x + 2, t)
