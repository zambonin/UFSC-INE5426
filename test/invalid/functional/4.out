[Line 7] semantic error: parameter x expected integer but received float
[Line 9] semantic error: function lambda has incoherent return type
[Line 9] semantic error: attribution operation expected integer array but received float array
